package.json

```diff
@@ -38,14 +39,14 @@
   },
   "devDependencies": {
     "@chromatic-com/storybook": "^1.3.1",
-    "@storybook/addon-essentials": "^8.0.6",
-    "@storybook/addon-interactions": "^8.0.6",
-    "@storybook/addon-links": "^8.0.6",
-    "@storybook/addon-onboarding": "^8.0.6",
-    "@storybook/blocks": "^8.0.6",
-    "@storybook/nextjs": "^8.0.6",
-    "@storybook/react": "^8.0.6",
-    "@storybook/test": "^8.0.6",
+    "@storybook/addon-essentials": "^8.0.8",
```

Comment:
need to rebase with dev or undo these changes, storybook dependencies shouldn;t need to be changed